Modules/ManualAssessment/classes/FileStorage/class.ilManualAssessmentFileStorage.php

```diff
@@ -0,0 +1,98 @@
+<?php
+require_once("Modules/ManualAssessment/interfaces/FileStorage/interface.ManualAssessmentFileStorage.php");
+include_once('Services/FileSystem/classes/class.ilFileSystemStorage.php');
+/**
+*
+* @author Stefan Hecken <stefan.hecken@concepts-and-training.de>
+*
+*/
+class ilManualAssessmentFileStorage extends ilFileSystemStorage implements ManualAssessmentFileStorage
+{
+	public function __construct($a_container_id = 0)
+	{
+		parent::__construct(self::STORAGE_WEB, true, $a_container_id);
+	}
+
+	protected function getPathPostfix()
```

Comment:
ILIAS needs them to create the order structure for saving the uploaded files in the webdir folder.